open_city_profile/graphene.py

```diff
@@ -70,3 +77,57 @@ def resolve(self, next, root, info, **kwargs):
             self.cached_loaders = True
 
         return next(root, info, **kwargs)
+
+
+TranslationLanguageType = graphql.GraphQLEnumType(
+    name="TranslationLanguage",
+    values=OrderedDict(
+        {
+            lang[0].upper(): graphql.GraphQLEnumValue(lang[0], description=lang[1])
+            for lang in settings.LANGUAGES
+        }
+    ),
+)
+
+
+TranslationDirective = graphql.GraphQLDirective(
+    name="translation",
```

Comment:
The GraphQL spec [recommends prefixing custom directive names](http://spec.graphql.org/October2021/#note-39239) with something. So what would a good prefix be? `hel_`? `helsinki_`?

Reply:
Ah. Didn't catch that. Looked at many custom directives and none used prefix. But yeah we should.

I suppose `hel_` could be ok?

Reply:
Short and concise :+1: